Products/Zuul/catalog/indexable.py

```diff
@@ -322,6 +331,50 @@ def idx_deviceOrganizers(self):
                 organizers.append( "/".join(sp[:-2]) )
         return organizers
 
+    @indexed(StringFieldType(stored=True, indexed=False), attr_query_name="tagNumber")
```

Comment:
Do we want to make the new fields indexed as well? Will we be searching on those fields?